pipelines/datasets/br_inmet_bdmep/utils.py

```
@@ -232,6 +237,156 @@ def get_clima_info(file: str) -> pd.DataFrame:
+    if sigla_uf not in valid_uf_codes:
+        raise ValueError(
+            f"UF inválida: {sigla_uf!r}. Deve ser uma sigla de estado brasileiro de duas letras."
+        )
+
+    if data_municipios is None:
+        df_municipios = bd.read_sql(
+            query="""
+            SELECT id_municipio, centroide
+            FROM `basedosdados.br_bd_diretorios_brasil.municipio`
+            WHERE sigla_uf = '{}'
+            """.format(sigla_uf.replace("'", "\\'")),
```

Ruff (0.15.12)

[error] Possible SQL injection vector through string-based query construction

(S608)